pdns/common_startup.cc

```diff
@@ -67,6 +67,7 @@ void declareArguments()
   ::arg().setSwitch("forward-dnsupdate","A global setting to allow DNS update packages that are for a Slave domain, to be forwarded to the master.")="yes";
   ::arg().setSwitch("log-dns-details","If PDNS should log DNS non-erroneous details")="no";
   ::arg().setSwitch("log-dns-queries","If PDNS should log all incoming DNS queries")="no";
+  ::arg().setSwitch("except-send-servfail","If PDNS should send ServFail on backends exception")="yes";
```

Comment:
maybe change this to "If a SERVFAIL should be sent on a backend exception, drop the query otherwise"